src/app/bucket/bucket.component.css

```diff
@@ -4,3 +4,4 @@
 .reorder-icon { vertical-align: middle; padding-right: 10px; padding-bottom: 15px; }
 .app-mat-list .mat-list-item { height: auto; min-height: 40px; }
 .mat-list-item app-objective { padding-bottom: 15px; }
+.width { width: 100%;}
```

Comment:
Let's not use such a generic class name as `width`, please. Maybe you can just add this `width: 100%` to the `.mat-list-item app-objective` line above?